.github/actions/docker-slim/action.yml

```diff
@@ -0,0 +1,24 @@
+name: Docker Slim
+description: Create slim versions of an existing container images
+
+inputs:
+  target:
+    description: The target image
+    required: true
+  tag:
+    description: The target image
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: Setup docker slim
+      run: |
+        mkdir /tmp/ds
+        cd /tmp/ds
+        curl -L -o ds.tar.gz https://github.com/slimtoolkit/slim/releases/download/1.40.11/dist_linux.tar.gz
```

Comment:
does a fixed version make sense? or could renovate handle that?

Reply:
Potentially. We will need to see, if Renovate will update this.